packages/components/src/Dropzone/sub-components/SelectedFileMessage/SelectedFileMessage.tsx

```diff
@@ -0,0 +1,50 @@
+import React from "react"
+import { Button } from "@kaizen/button"
+import { Icon } from "@kaizen/component-library"
+import selectedFileSvgIcon from "@kaizen/component-library/icons/file.icon.svg"
+import acceptedFileSvgIcon from "@kaizen/component-library/icons/success.icon.svg"
+import { Paragraph } from "@kaizen/typography"
+import { trunicateFilename } from "../../utils"
+import { Hyperlink } from "../Hyperlink/Hyperlink"
+
+interface SelectedFileMessageProps {
+  fileName: string
+  handleOpenDialog: () => void
+}
+
+export const AcceptedFileIcon = (): JSX.Element => (
+  <span className="inline-block mb-8 text-green-500">
```

Comment:
I noticed all your icons are wrapped in an `mb-8` class. Instead of doing this individually, you should add a `div` container in the `Dropzone.tsx` to house all the icons:

```
<div className="mb-8">
    <DropzoneIcon />
</div>
```

This way the overall content structure is easily manipulated (eg. changing the spacing between icons and content).